api/src/pcapi/routes/pro/collective_offers.py

```diff
@@ -203,6 +203,12 @@ def edit_collective_offer(
         offers_api.update_collective_offer(offer_id=dehumanized_id, new_values=body.dict(exclude_unset=True))
     except offers_exceptions.SubcategoryNotEligibleForEducationalOffer:
         raise ApiErrors({"subcategoryId": "this subcategory is not educational"}, 400)
+    except offers_exceptions.OfferUsedOrReimbursedCantBeEdit:
+        raise ApiErrors({"offer": "Les offres utilisées ou remboursées ne sont pas modifiable."}, 403)
```

Comment:
```suggestion
        raise ApiErrors({"offer": "Les offres utilisées ou remboursées ne sont pas modifiables."}, 403)
```

Est-il prévu de décider d'une langue pour les messages d'erreur de cette route ? Actuellement, on a des messages en anglais et d'autres en français.